### ISSUE TITLE
Not working on Ubuntu Precise

### ISSUE THREAD

**Issue by d-ph** [thias/puppet-vsftpd#7]
Hey Matthias,

in short: Ubuntu (tested in Precise) uses `smbd` as the Samba service name. In `manifests/params.pp` it is set as: `samba` resulting in puppet not being able to restart the service (unknown service error). I've edited:

```
case $osfamily {
    'RedHat': { $service = [ 'smb', 'nmb' ] }
    'Debian': { $service = [ 'samba' ] }
     default: { $service = [ 'samba' ] }
  }
```

to:

```
case $osfamily {
    'RedHat': { $service = [ 'smb', 'nmb' ] }
    'Debian': { $service = [ 'smbd' ] }
     default: { $service = [ 'smbd' ] }
  }
```

and it worked. Note that I was too lazy to check, whether I should change only the `'Debian'` or both `'Debian'` and `default` - I just changed both. Note that on Debian the service is named: `samba`, so it looks like there should be another key `ubuntu` specifically for this issue.

Cheers,
d-ph

**Comment by thias** [comment]
Hmm, I'm guessing you wanted to report this against `puppet-samba`, right? Do let me know if I have a wrong link somewhere that led you here.

The solution should be to switch to `$::operatingsystem` in the `case` statement or (to not have to list all members of the RedHat family) to just add another `case` inside of the `Debian` section.

**Comment by thias** [comment]
This should be fixed in git and in the 0.1.2 release on the forge.

**Comment by d-ph** [comment]
You're right, I wanted to make an issue report to the `puppet-samba` module. I just had a lot of browser bookmarks opened and it ended in a wrong place.

0.1.2. works for me now. Thanks
